index-handlers/src/test/java/no/sikt/nva/nvi/index/OpenSearchClientTest.java

```diff
@@ -162,13 +195,65 @@ void shouldReturnAggregationsWithExpectedCount(Entry<String, Integer> entry)
                             documentFromString("document_rejected_collaboration.json"));
 
         var searchResponse =
-            openSearchClient.search("*", NO_FILTER, USERNAME, YEAR, CUSTOMER,
+            openSearchClient.search(null, NO_FILTER, USERNAME, YEAR, CUSTOMER,
                                     DEFAULT_OFFSET_SIZE, DEFAULT_QUERY_SIZE);
         var docCount = getDocCount(searchResponse, entry.getKey());
 
         assertThat(docCount, is(equalTo(entry.getValue())));
     }
 
+    @Test
+    void shouldReturnSearchResultsWithContributorOfSearchedInstitution()
+        throws IOException, InterruptedException {
+        addDocumentsToIndex(documentFromString("document_with_contributor_from_ntnu.json"),
+                            documentFromString("document_with_contributor_from_sikt.json"),
+                            documentFromString("document_with_contributor_from_sikt_but_not_creator.json")
+        );
+
+        var siktInstitutionId = "https://api.dev.nva.aws.unit.no/cristin/organization/20754.0.0.0";
+        var searchResponse =
+            openSearchClient.search(siktInstitutionId, "", USERNAME, YEAR, CUSTOMER,
+                                    DEFAULT_OFFSET_SIZE, DEFAULT_QUERY_SIZE);
+
+        assertThat(searchResponse.hits().hits(), hasSize(1));
+    }
+
+    @Test
+    void shouldReturnSearchResultsWithContributorOfSearchedInstitutionWhenNotSearchingTopLevelInstititution()
+        throws IOException, InterruptedException {
+        addDocumentsToIndex(documentFromString("document_with_contributor_from_ntnu.json"),
+                            documentFromString("document_with_contributor_from_sikt.json"),
+                            documentFromString("document_with_contributor_from_sikt_but_not_creator.json")
+        );
+
+        var siktInstitutionId = "https://api.dev.nva.aws.unit.no/cristin/organization/194.0.0.0";
```

Comment:
Not sure if I misunderstood the test, but should the affiliation you are searching for not be `"https://api.dev.nva.aws.unit.no/cristin/organization/194.0.0.1"`? Also, rename it to `ntnuSubUnitId`? 
```
var ntnuSubUnitId = "https://api.dev.nva.aws.unit.no/cristin/organization/194.0.0.1";
        var searchResponse =
            openSearchClient.search(ntnuSubUnitId, "", USERNAME, YEAR, CUSTOMER,
                                    DEFAULT_OFFSET_SIZE, DEFAULT_QUERY_SIZE);
```

Reply:
Also, I think I would remove the other documents from the test to make sure I am testing the right thing:
```
@Test
    void shouldReturnSearchResultsWithContributorOfSearchedInstitutionWhenNotSearchingTopLevelInstititution()
        throws IOException, InterruptedException {
        addDocumentsToIndex(documentFromString("document_with_contributor_from_ntnu.json"));

        var ntnuSubUnitId = "https://api.dev.nva.aws.unit.no/cristin/organization/194.0.0.1";
        var searchResponse =
            openSearchClient.search(ntnuSubUnitId, "", USERNAME, YEAR, CUSTOMER,
                                    DEFAULT_OFFSET_SIZE, DEFAULT_QUERY_SIZE);

        assertThat(searchResponse.hits().hits(), hasSize(1));
    }
```